### PR TITLE
CONTRIB/JENKINS: Remove the coverity dir if the coverity check passed

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9721/ for details (Mellanox internal link).

**Comment by alinask** [comment]
@yosefe please review.

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -953,7 +953,8 @@ run_coverity() {
 			cov-format-errors --dir $cov_build --emacs-style
 			echo "not ok 1 Coverity Detected $nerrors failures # $cov_url" >> coverity.tap
 		else
-		echo "ok 1 Coverity found no issues" >> coverity.tap
+			echo "ok 1 Coverity found no issues. deleting the coverity directory" >> coverity.tap
```
Comment (yosefe) [inline contrib/test_jenkins.sh]: "deleting the coverity directory" should not be printed to coverity.tap, only to log file

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9725/ for details (Mellanox internal link).

**Pull request merged**